static/app/views/settings/organization/navigationConfiguration.tsx

```diff
@@ -109,6 +110,17 @@ const organizationNavigation: NavigationSection[] = [
         id: 'early-features',
         recordAnalytics: true,
       },
+      {
+        path: `${organizationSettingsPathPrefix}/dynamic-sampling/`,
+        title: t('Dynamic Sampling'),
+        description: t('Manage your sampling rate'),
+        show: ({organization, isSelfHosted}) =>
+          !!(
+            organization &&
+            hasDynamicSamplingCustomFeature(organization) &&
+            isSelfHosted
```

Comment:
We only show the link in the org menu section if the instance is self hosted.
Otherwise (on SaaS) it will be in the `Usage & Billing` section -> https://github.com/getsentry/getsentry/pull/15383